UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/AzureBlobStorageClient.cs

```diff
@@ -32,5 +32,13 @@ public async Task<string> DownloadTextAsync(CloudBlockBlob cloudBlockBlob)
         {
              return await cloudBlockBlob.DownloadTextAsync();
         }
+        public async Task<bool> DeleteFileFromContainer(string storageAccountConnectionString, string containerName, string batchId)
```

Comment:
Remove this method as you can reuse existing method GetCloudBlockBlob of same class.
Get the blob and then delete.

Reply:
As per suggestion, i have done required changes and removed this method.